pkg/trace/pb/hook_test.go

```diff
@@ -12,8 +12,8 @@ import (
 	"github.com/tinylib/msgp/msgp"
 )
 
-func TestMetaHook(t *testing.T) {
-	t.Run("off", func(t *testing.T) {
+func TestMetaHooks(t *testing.T) {
+	t.Run("meta off", func(t *testing.T) {
```

Comment:
We only use one single word for tests throughout. Not multiple words with spaces. Example names for your tests:
* `off`
* `on`
* `struct`
* `meta`
etc.